src/tests/auth/userVerification.test.js

```diff
@@ -0,0 +1,37 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../server';
+import { user } from '../fixtures/user.fixture';
+import TokenService from '../../services/token.service';
+
+const token = TokenService.generateToken({
+	email: user.email
+});
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('/GET email verification', () => {
+	it('Should create a new user tha is going to be verified ', (done) => {
```

Comment:
Why are you creating another test of creating a user? instead, you can use the `before hook` function to create a new user

Reply:
Done.